linux/src/file_util_linux.c

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include <string.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <inttypes.h>
+#include <stdbool.h>
+ #include <unistd.h>
+
+#include "c_pal/gballoc_hl.h" // IWYU pragma: keep
+#include "c_pal/gballoc_hl_redirect.h"
+
+#include "c_logging/logger.h"
+
+#include "c_pal/windows_defines.h"
+
+#include "c_pal/file_util.h"
+
+typedef struct CREATE_FILE_LINUX_TAG
+{
+    int h_file;
+
+} CREATE_FILE_LINUX;
+
+//Codes_SRS_FILE_UTIL_LINUX_09_002: [ file_util_create_file shall allocate memory for the file handle. file_util_create_file will succeed and return a non-NULL value. ]
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

"Codes_...." lines are usually wrapped in "/* */".

Use block comments for requirements instead of line comments.